ff2mpv.js

```diff
@@ -2,6 +2,17 @@ function ff2mpv(url) {
     browser.runtime.sendNativeMessage("ff2mpv", { url: url });
 }
 
+function notifyContentScript(e) {
+    browser.tabs.query({active: true, currentWindow: true}, function(tabs){
+        browser.tabs.sendMessage(tabs[0].id, {launch: true});
+    });
+}
+
+browser.contentScripts.register({
+    matches: ["<all_urls>"],
```

Comment:
Since the extension works with more than just yt, I decided to go with any url instead of maintaining a list of working video streaming sites.

Reply:
Makes sense. Good call.